src/styles/_catalog.scss

```
@@ -1,22 +1,151 @@
-.catalog {
+.main__block {
   display: flex;
-  flex-wrap: wrap;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin-bottom: 46px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.card {
+  width: $cell-size;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s, color 0.3s, box-shadow 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    color: #34568b;
+    box-shadow: #00acdc3a 0 0 16px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
 
-  &__card {
+  &__description {
     display: flex;
     justify-content: center;
+  }
+
+  &__title {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
```

```suggestion
    font-family: "Roboto", sans-serif;
    font-style: normal;
```

You repeat this part six times, maybe add it to mixins? like '@include font'?